reference-data-proxy/src/v1/controllers/estore/eStoreApi.controller.ts

```diff
@@ -11,7 +11,7 @@ const postToEstore = async (apiEndpoint: string, apiData: any) => {
     return false;
   }
 
-  console.log(`Calling eStore API (/${apiEndpoint})`);
+  console.info(`Calling eStore API (/${apiEndpoint})`);
```

Comment:
does printing the endpoint show the full url? and isn't the url under an .env so kept 'secret'?

Reply:
Nope just the endpoint, not the URL. Also they are in logs which can only be accessed by us.